jest.config.js

```diff
@@ -1,3 +1,10 @@
-const commonConfig = require('./build/jest/jest.config.common')
+const { rollup: lernaAlises } = require('lerna-alias')
 
-module.exports = commonConfig
+// packages/react/src -> packages/react,
+// as lernaAliases append 'src' by default
+const stardustPackages = lernaAlises({ sourceDirectory: false })
+const projects = Object.keys(stardustPackages).map(packageName => stardustPackages[packageName])
+
+module.exports = {
+  projects,
```

Comment:
Reused [`projects`](https://jestjs.io/docs/en/configuration#projects-array-string-projectconfig) feature, now we can have different configs for each package ✋